### ISSUE TITLE
Mutes ads, does not remove them (and now it's actually muted everything)

### ISSUE THREAD

**Issue by zefir-git** [SecUpwN/Spotify-AdKiller#124]
It mutes the ads (not using the Spotify controls) instead of skipping them. Banners, etc. are still visible.

If you close the app while a muted ad is playing, everything remains muted, all songs! Any info on how to fix that would be highly appreciated!

**Comment by zefir-git** [comment]
Killing the adkiller process got my songs unmuted.

**Comment by Darguima** [comment]
Spotify-AdKiller only silences the ads, it doesn't work as an ad blocker (so you'll still see the banners).
But you can set it to play local tracks while the ad is muted by setting `CUSTOM_MODE` and downloading songs to `CUSTOM_MUSIC`. You can read more about [here](https://github.com/SecUpwN/Spotify-AdKiller#settings)

About Spotify remains muted, you already tried [this](https://github.com/SecUpwN/Spotify-AdKiller#important-notes)?

**Comment by zefir-git** [comment]
I tried about 2 or 3 different solutions to remove ads as well but was left with the impression they did not work (maybe too outdated). I tried this one and it did work for removing ads completely, or at least, as far as I remember, until the Spotify client was updated, which I would guess was a major update as many features were updated.
I cannot recall precisely as this was almost a year ago and I have since switched to using YouTube using a browser ad-block extension which works exceptionally well in removing all ads.
As I think about it, not sure if your repo is what I ended up using, might have been one of the other methods that did not work for me upon my initial tests. I do understand that it probably matters a lot whether the Spotify client was installed via snap or as a deb package and it's probably not easy to interfere with it to block the ads and therefore this issue might not be relevant, especially if you have made updates to your software since I used it. Just stating this here for any potential readers to know that I cannot back up my initial claims in this issue and this shall by no means be considered a negative review of any sort.
